trackingC.Rmd

```diff
@@ -51,6 +51,12 @@ lognorm <- function(m, sd){
 
 N_RUNS <- 1000
 
+# We use GitHub Actions to make sure this RMarkdown knits successfully
+# But if running there, only do a small number of Hector simulations
+if(Sys.getenv("CI") == "true") {
```

Comment:
Is this checking that if in a runner environment, decrease the number of runs? (What does `..."CI" == "true"`) refer to)

Reply:
Exactly—if in a runner, don't bother with lots of runs. We just want to ensure the Rmarkdown knits.

Line 56 gets what is called an _environment variable_ (a setting in the system that's running this script). GitHub Actions will also set a variable "CI" as "true".